web/README.md

```diff
@@ -0,0 +1,7 @@
+Web
```

Comment:
Wow, this one is sparse. I should take a crack at adding some more 😬

Reply:
Yeah -- there are a few like this. Once all the guides are at the root level and have consistent formatting, I want to audit them all for guides that need improvement.

Reply:
Now we have #616 to track this 🙂